<launch>
    <node name="serial_node" pkg="rosserial_python" type="serial_node.py" output="screen">
        <param name="port" value="/dev/arduino"/>
    </node>
    <node name="arduino_node" pkg="lowlevel_arduino" type="arduino_node.py" output="screen">
        <param name="rate" value="20"/>
    </node>
</launch>
